src/components/InjectedComponents/DecryptedPost.tsx

```diff
@@ -161,6 +164,9 @@ export function DecryptPost(props: DecryptPostProps) {
         undefined,
     )
 
+    const postPayload = deconstructPayload(encryptedText, null)
```

Comment:
Here is the hoisted variable in order to evaluate it only once in single rendering task.

Reply:
According to my knowledge this seems the same as before... If you need to keep it between re-renders you need to wrap it with `useState`

Reply:
Updated to using `useState`